src/main/java/com/softserveinc/task03/Country.java

```diff
@@ -0,0 +1,92 @@
+package com.softserveinc.task03;
+
+import com.softserveinc.task02.City;
+import java.util.Arrays;
+
+public class Country {
+    private String name;
+    private String continent;
+    private String code;
+    private City capital;
+    private City[] cities;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getContinent() {
+        return continent;
+    }
+
+    public void setContinent(String continent) {
+        this.continent = continent;
+    }
+
+    public String getCode() {
+        return code;
+    }
+
+    public void setCode(String code) {
+        this.code = code;
+    }
+
+    public City getCapital() {
+        return capital;
+    }
+
+    public void setCapital(City capital) {
+        this.capital = capital;
+    }
+
+    public City[] getCities() {
+        return cities;
+    }
+
+    public void setCities(City[] cities) {
+        this.cities = cities;
+    }
+
+    public Country(String name, String continent, String code, City capital) {
+        this.name = name;
+        this.continent = continent;
+        this.code = code;
+        this.capital = capital;
```

Comment:
You have to initialize the field cities
`this.cities = new City[] {capital};`

Reply:
add